conductor.py

```diff
@@ -985,6 +689,99 @@ def log_time():
        states_file.write('\n'+str(tuple((datetime.now().isoformat(), j))))
    return ''
 
+# JSON file upload function to allow first .json upload and check text format 
+# It checks "id:" "title:" and "candidates:"
+@app.route('/upload-ballot-manifest-json', methods=['POST'])
```

Comment:
This isn't a ballot manifest, so a better name might be something like `'/upload-all-contests-data'`